week-1-practise/level-1/8-function-output-1.js

```diff
@@ -7,6 +7,10 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+//generate random numbers everytime it's called, from 0 to one (not included)
 // - How many parameters does the function take?
+//none needed as random() will generate the number alone
 // - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+//getNumber
+// - Where is the function called? with what parameters?
+//line 6 with console log witout any parameters
```

Comment:
Close! The _result_ is _used_ on line 6, but the function is not _called_ on that line (like in exercise 1).